README.md

```diff
@@ -36,14 +38,44 @@ Message: "WARNING: The pod myapp-xyz-1234 in namespace default is running with
 sidecar proxy version 0.2.10 but your environment is running Istio
 version 0.2.12. Consider upgrading the sidecar proxy in the pod."
 ```
+<br>
+<br>
 
-## Running
+## Running Istio-Vet
 
-The official docker image is `quay.io/aspenmesh/istio-vet:master`
+You can build and run Istio-Vet from this repo, or use the docker image (locally or from within in a kubernetes cluster).
 
-Container image can be deployed in a kubernetes cluster or run locally.
 
-### Local
+### Building Istio-Vet
+
+If you want to build Istio-Vet in order to contribute, please see the instructions for [Contributors.](https://github.com/aspenmesh/istio-vet#contributing)
+
+#### Installation
+
+* From within your Go environment, navigate to the directory where you want Istio-Vet to be installed.
+* Make sure you have Go installed in your Path:
+    ```bash
+    export GOPATH=$HOME/gocode
+    export PATH=$PATH:$GOPATH/bin
+    ```
+* Run the go command to get Istio-Vet:
+    ```bash
+    go get github.com/aspenmesh/istio-vet/cmd/vet
```

Comment:
Just add a link here which refer to the Contributing section for building istio-vet locally, then we can remove this.